### ISSUE TITLE
How To Get Index of GroupedListView?

### ISSUE THREAD

**Issue by zgramming** [Dimibe/grouped_list#21]
Thank's for making this great package. I have question how to get index of this GroupedListView ? It's Possible ? 

 I check inside **itemBuilder** not have **index** property , and i check too inside your file not see **index** property

### Your File
```
itemBuilder: (context, index) {
              int actualIndex = index ~/ 2;
              if (index == 0) {
                if (widget.useStickyGroupSeparators) {
                  return buildEmptySeparator(actualIndex);
                }
                return _buildGroupSeparator(actualIndex);
              }
              if (index.isEven) {
                E curr = widget.groupBy(_sortedElements[actualIndex]);
                E prev = widget.groupBy(_sortedElements[actualIndex - 1]);
                if (prev != curr) {
                  return _buildGroupSeparator(actualIndex);
                }
                return widget.separator;
              }
              return widget.itemBuilder(context, _sortedElements[actualIndex]);
            },
```

What i expected is , i want display number order in this List. How can i achieved this ?

<img src="https://user-images.githubusercontent.com/38829404/77995386-e6370000-7355-11ea-8bd4-437d0cbc8d39.jpeg" height="500">

## My Source Code
```
SizedBox(
                          height: sizes.height(context) * .50,
                          child: GroupedListView<PencatatanAirModel, String>(
                            elements: waterProvider.listByMonthAndYear,
                            groupBy: (model) => model.lantai,
                            separator: Divider(),
                            groupSeparatorBuilder: (value) => Padding(
                              padding: const EdgeInsets.all(4.0),
                              child: Text(
                                value,
                                style: TextStyle(
                                  fontWeight: FontWeight.bold,
                                ),
                              ),
                            ),
                            itemBuilder: (_, PencatatanAirModel result) =>
                                InkWell(
                              onTap: () {
                                Navigator.of(context).pushNamed(
                                    WaterRecordDetailScreen.routeNamed);
                                waterProvider.setListLevel3(
                                  idDenah: result.idDenah,
                                );
                              },
                              child: Padding(
                                padding:
                                    const EdgeInsets.symmetric(vertical: 4.0),
                                child: Row(
                                  children: <Widget>[
                                    // titleHeaderRow(title: '$orderIndex'),
                                    titleHeaderRow(title: result.noPelanggan),
                                    titleHeaderRow(title: result.area),
                                    titleHeaderRow(title: result.meteran),
                                  ],
                                ),
                              ),
                            ),
                          ),
                        ),
```

**Comment by Dimibe** [comment]
Currently it's not possible to get the index information. Maybe I will add an option in a later release.

**Comment by Dimibe** [comment]
Implemented within #24. Will be available in the release 2.3.0. Have a look into the readme for usage.
